### PR TITLE
UI changes#2

### PR DESCRIPTION
**Pull request by maks-taraniuk**

### CODE REVIEW

File: app/src/main/res/layout/main_activity.xml [hunk 1]
```diff
@@ -23,10 +23,10 @@
         <EditText
             android:id="@+id/textView"
             style="@style/TextNumber.Input"
-            android:layout_width="0dp"
+            android:layout_width="@dimen/dimen_zero"
             android:layout_height="wrap_content"
             android:layout_marginTop="30dp"
```
Comment (DmitriyGlazunov) [inline app/src/main/res/layout/main_activity.xml]: No need to use hardcoded values in your xml. Try to move this value to resources.

File: app/src/main/res/layout/main_activity.xml [hunk 2]
```diff
@@ -23,10 +23,10 @@
         <EditText
             android:id="@+id/textView"
```
Comment (DmitriyGlazunov) [inline app/src/main/res/layout/main_activity.xml]: Actually it's not textView, it's EditText. Rename this id to inputET.

File: app/src/main/res/layout/main_activity.xml [hunk 3]
```diff
@@ -39,12 +39,11 @@
             android:layout_height="wrap_content"
             android:layout_marginTop="10dp"
             android:layout_marginBottom="5dp"
```
Comment (DmitriyGlazunov) [inline app/src/main/res/layout/main_activity.xml]: The same situation here, move this values to resources.

File: app/src/main/res/layout/main_activity.xml [hunk 4]
```diff
@@ -70,6 +69,7 @@
             android:layout_height="0dp"
```
Comment (DmitriyGlazunov) [inline app/src/main/res/layout/main_activity.xml]: This value and others should be taken from resources.

File: app/src/main/res/layout/main_activity.xml [hunk 5]
```diff
@@ -121,7 +123,7 @@
             app:layout_constraintBottom_toTopOf="@+id/button"
             app:layout_constraintEnd_toStartOf="@+id/button12"
```
Comment (DmitriyGlazunov) [inline app/src/main/res/layout/main_activity.xml]: Please rename all buttons id's to correct name. "button2" is not good idea for name id.

File: app/src/main/res/values/colors.xml [hunk 6]
```diff
@@ -8,4 +8,17 @@
     <color name="colorText">#262727</color>
 
     <color name="colorBackground">#1E1E1E</color>
+
+    <!-- click color -->
+    <color name="startClick_color">#5E5A63</color>
```
Comment (DmitriyGlazunov) [inline app/src/main/res/values/colors.xml]: Need to change this name to 'start_click_color'

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -8,4 +8,17 @@
...
+    <color name="endClick_color">#B5B5B5</color>
+    <!-- buttons numbers -->
+    <color name="startColor_number_button">#FEFEFF</color>
+    <color name="endColor_number_button">#8C8A8A</color>
+    <!-- buttons numbers -->
```
Comment (DmitriyGlazunov) [inline app/src/main/res/values/colors.xml]: Actually it's not numbers, it's colors

**Comment by maks-taraniuk** [comment]
Fixed all the tasks

**Pull request merged**